### DISCUSSION TITLE
对于同应用名进程运行时长进行合并，或对相似程序目录运行时长进行合并

### DISCUSSION THREAD

**Discussion by Ovler-Young** [Planshit/Tai#27] [Ideas]
![image](https://user-images.githubusercontent.com/44089074/148636102-1bcab3d1-595e-4a5b-b603-0c198317c8e5.png)
如logseq等软件更新后运行目录会改变，但都有规律，如 `C:\Users\YU\AppData\Local\Logseq\app-0.5.8\Logseq.exe` 和 `C:\Users\YU\AppData\Local\Logseq\app-0.5.7\Logseq.exe`，能否设置规则将这类软件时长合并。或者，将同应用名进程运行时长进行合并也可以解决问题。

**Comment by noberumotto** [comment]
收到，下个版本将会处理这个问题。

在开发之前考虑到可能会出现两个不同软件进程名相同的情况，所以把进程路径也作为唯一标识符的一部分，现在看来似乎出现这种情况的概率是很小的。

**Reply by noberumotto** [reply]
已经更新，请升级版本，[1.0.0.2](https://github.com/Planshit/Tai/releases/tag/1.0.0.2)
